ScoutSuite/frontend/src/pages/Resources/ResourcePartialWrapper/index.js

```diff
@@ -0,0 +1,57 @@
+import React, { Suspense } from 'react';
```

Comment:
so much of this file is similar to the `SelectedItemContainer`, do you think there's a way to make it more generic for the two

Reply:
The logic will be different when I will actually load the partials for the resources (still waiting on the backend change)